src/data/ACTIONS/root/DNC.ts

```diff
@@ -8,41 +8,71 @@ export const DNC = ensureActions({
 		name: 'Single Standard Finish',
 		onGcd: true,
 		cooldown: 1500,
+		statusesApplied: [
+			'STANDARD_FINISH',
+			'STANDARD_FINISH_PARTNER',
+			'ESPRIT',
+		],
```

Comment:
As much as it's technically wrong data, it's probably better if you only put the statusesApplied on the "got all steps" finishes - these are basically here to feed precast synthing, and the adapter for prepull statuses implicitly assumes that only one action can add a given status -- behavior isn't guaranteed deterministic if multiple actions have the same statusesApplied (though it will generally pick the "first" one in the status array, which is generally going to be the first one defined in the data files).

Reply:
Fair enough, I wasn't _entirely_ sure what the statusesApplied field was actually being used for so I was trying to be as complete as I could. If that's all it's mainly for then I agree we can dispense with those (and the data for Tillana too).